scripts/ci-entrypoint.sh

```diff
@@ -107,19 +122,11 @@ run_upstream_e2e_tests() {
         fi
     fi
 
-    # get the number of worker nodes
-    NUM_NODES="$(kubectl get nodes --kubeconfig="$KUBECONFIG" \
-    -o=jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.spec.taints}{"\n"}{end}' \
-    | grep -cv "node-role.kubernetes.io/master" )"
-
-    # wait for all the nodes to be ready
-    kubectl wait --for=condition=Ready node --kubeconfig="$KUBECONFIG" --all || true
```

Comment:
We can remove this with the addition of https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/659/files#diff-4bc72f6db52f65045884231f54b3e802R173-R174.